### ISSUE TITLE
Chapter 11, Questions 1,2

### ISSUE THREAD

**Issue by eyalovadya-cargozone** [johnnyb/efbsolutions#3]
The solution does not match the answer given in the explenation (the correct solution).
![Screenshot_20220312-231841_Samsung Notes](https://user-images.githubusercontent.com/75927757/158035356-c5d63f23-ab76-4d9b-9e90-4884ad8ab28e.jpg)

**Comment by DayControls** [comment]
Yes, same issue with the first question in this chapter (11) review.  It says the solution is 9.4mA and then after explaining it, it says the solution is 5.93mA

**Comment by johnnyb** [comment]
Sorry - I get so many git notifications from my job I didn't even notice people were commenting on this!  I'll try to get back up to date this weekend on this.

**Comment by evanbuschor** [comment]
I think the solution is using a 9v bat for the source voltage and the actual question is referencing a 5v source.
